playbooks/roles/mysql/tasks/main.yml

```diff
@@ -3,7 +3,7 @@
 # edX is currently using MySQL 5.6, which is not available on Xenial by default.
 - name: Add PPA for MySQL 5.6
   apt_repository:
-    repo: "ppa:ondrej/mysql-5.6"
+    repo: "deb http://archive.ubuntu.com/ubuntu trusty universe"
```

Comment:
@farhaanbukhsh This repo will not work unfortunately.
Our servers use MySQL 5.6.35, and we cannot downgrade minor versions.

Reply:
@farhaanbukhsh Can you look at another PPA or Ansible Galaxy playbook to install the right MySQL version? An upgrade is fine, but a downgrade is not.

Reply:
sure I a having a look, mostly trusty stopped at 5.6.33 and there are not there roles that I could see supporting 5.6, most of the people were using same PPA as us and as it stopped they stopped supporting the version. But I am trying to dig in and see if I can find a PPA or a way to get 5.6.35 or above.